<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cole</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>631</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Registrar alumno</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Nombre">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de nacimiento:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Curso:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Curso">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DNI:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Dni">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblResultados">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Apellido">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Division:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Division">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>350</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FechaNac">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab1">
    <attribute name="title">
     <string>Cuotas del alumno</string>
    </attribute>
    <widget class="QLabel" name="nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DNI:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuotas que debe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuotas que pag√≥:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio de las cuotas:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PC">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDni">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_continuar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>340</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="D">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="P">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="lista">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>290</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>290</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Info sobre alumno</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DNI:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDni2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>321</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mostrar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>360</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lista2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>401</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Nombre</tabstop>
  <tabstop>Apellido</tabstop>
  <tabstop>FechaNac</tabstop>
  <tabstop>Curso</tabstop>
  <tabstop>Division</tabstop>
  <tabstop>Dni</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>cmbDni</tabstop>
  <tabstop>D</tabstop>
  <tabstop>P</tabstop>
  <tabstop>PC</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>btn_continuar</tabstop>
  <tabstop>cmbDni2</tabstop>
  <tabstop>btn_mostrar</tabstop>
  <tabstop>lista2</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lista</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_continuar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>aceptar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>guardar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>guardar2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>limpiar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_continuar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>limpiar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbDni2</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>mostrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_mostrar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>limpiar2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbDni</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>ver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aceptar()</slot>
  <slot>buscar()</slot>
  <slot>continuar()</slot>
  <slot>guardar()</slot>
  <slot>mostrar()</slot>
  <slot>guardar2()</slot>
  <slot>ver()</slot>
  <slot>limpiar()</slot>
  <slot>limpiar2()</slot>
 </slots>
</ui>
